reference_entities/configure_entity_limits.rst

```diff
@@ -0,0 +1,12 @@
+Configure Entity Limits
+=======================
+
+For the Reference Entities feature (*introduced in 3.0*), some limits about number of entities have been defined to guarantee that the PIM is functional and runs smoothly.
+However, as your catalog is very unique, **you may need to raise those values**, this chapter explains how.
```

Comment:
optional: maybe add a small comment to say that if they hit those limits, it's maybe because they need to change their modelisation. I don't know if we should do it here, I'm just suggesting :)

Reply:
Could we add that if they raise these limits we won't support them and guarantee that the PIM is functional and runs smoothly and that they need to test the PIM with their customized limits?